### DISCUSSION TITLE
Settings tab missing from repository after transferring ownership

### DISCUSSION THREAD

**Discussion by le-michael** [community/community#22222] [New to GitHub]
<p>Hi,</p>
<p>I am working on a project with some peers and we transferred the ownership of the repository to my account yesterday. However, after transferring the settings tab no longer shows up on the repository even though I am the owner of it now.</p>
<p>Is this a common issue? Is there a way to get this fixed?</p>

**Comment by bgpat** [comment, marked as answer]
<p>In my account, the <code>Settings</code> tab is missing from the UI of some repositories, even though without transferring. Although, I can access it by adding <code>/settings</code> to the URL.</p>

**Comment by le-michael** [comment]
<p>This worked perfectly thanks</p>

**Reply by abhinavsalar04** [reply]
I got into same problem but I am not able to access settings tab, either by this method or directly

**Comment by ppreshant** [comment]
I tried adding `/settings` to the URL but I get the 404 error :frowning_face: 

It might be something to do with restrictions that github has when transferring a private repo from a free account?

This is the message I see before accepting transfer.
> if @username is using GitHub Free and accepts the transfer, they will lose access to private repository features:

    Code owners
    Any existing wikis
    Pulse, Contributors, Community, Traffic, Commits, Code Frequency, Network, and Forks on the Insights tab
    Draft PRs
    Multiple assignees for issues and PRs
    Multiple reviewers for PRs
    Branch and tag protection rules

> @username can [upgrade](https://github.com/pricing) their plan before accepting the transfer to avoid losing access.

For now I am just going to create a fresh private repo in the organization and push the repo there (_only works since I don't have any issues/other github things on the repo_)
